src/Features/LanguageServer/ProtocolUnitTests/Completion/CompletionFeaturesTests.cs

```diff
@@ -540,6 +540,10 @@ void M()
         itemFromNS1.Data = completionResult.ItemDefaults.Data;
         itemFromNS2.Data = completionResult.ItemDefaults.Data;
 
+        // Remove the label details as this is the behavior of the VSCode client when resolving completion items.
+        itemFromNS1.LabelDetails = null;
```

Comment:
ah this is why the unit tests didn't catch it?

Reply:
Yea. And I blame jetlag as the reason why it's not caught by manual testing